### ISSUE TITLE
is there a way to change to seconds?

### ISSUE THREAD

**Issue by fokeiro2020** [noelvissers/site-watcher#1]
gm. scripts working great but is there a way to turn minutes into seconds instead?

**Comment by noelvissers** [comment]
Yes, this is possible. The cron package used accepts seconds as an parameter. I'm not sure if this will function correctly if multiple sites are added, and checking sites takes more than the selected time, so ill have to test this. Ill get back to this when I have time
